Add studio version in header for local self host

joshenlim opened the pull request:
Strategy is that Studio gets the version its on by an environment variable - the UI changes here is set up such that is the env var is present, we'll then render the Version popover

We'll do this by specifying a BUILD_VERSION env var when building the docker image - so this also needs to be updated in [here](https://github.com/supabase/supabase/blob/master/.github/workflows/publish_image.yml) eventually too which I might need @sweatybridge's help on this one as my Docker knowledge is not the best 😅 (I might be missing a detail or two too while i'm trying to piece what needs to be done in my mind 😬)

Nonetheless, verified working locally by building docker with the Dockerfile in studio via
`docker build . -f apps/studio/Dockerfile --target production -t studio:latest --build-arg BUILD_VERSION="20250219-7101314"`

Changes should not affect the hosted version.

On latest version:
![image](https://github.com/user-attachments/assets/5ddca0de-7129-4be5-9e11-ac3956a07d46)

Not on latest version:
![image](https://github.com/user-attachments/assets/de9aee6a-8586-4009-a45d-39ebd0201bb6)

Release schedule modal:
Approximate release date is based on last image release + 14 days (2 weeks)
![image](https://github.com/user-attachments/assets/adfe7e54-bc27-406e-b4c0-4b3570134dc1)


Addresses https://linear.app/supabase/issue/FE-390/show-warning-on-localself-hosted-when-cli-version-is-out-of-date

joshenlim committed:
Auth Settings IA Update - Sign In / Up (#33579)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* Smol CSS tweak

---------

Co-authored-by: Saxon Fletcher <saxonafletcher@gmail.com>

kamilogorek committed:
docs: Add deprecated label to mgmt api docs (#33734)

joshenlim committed:
Tiny nit advanced auth form (#33740)

kevcodez committed:
feat: break down log drain egress (#33763)

fsansalvadore committed:
feat: docs InfoTooltip (#33654)

* docs InfoPopUp

* docs InfoTooltip

* Add mobile sheet for InfoTooltip

* add InfoTooltip to Info Tooltip contributing guide

* keyboard and screen reader behavior

* remove example

* remove hover trigger on mobile

---------

Co-authored-by: Charis Lam <26616127+charislam@users.noreply.github.com>

kevcodez committed:
chore: only include platform/v1 routes in api types (#33753)

joshenlim committed:
Add studio version in header for local self host

saltcod committed:
Chore/delay disk size report (#33686)

Dont show warning if project coming_up

joshenlim committed:
Remove manual pause on auth e2e test (#33762)

sweatybridge commented on apps/studio/Dockerfile:
```diff
@@ -41,6 +41,9 @@ COPY --from=turbo /app/out/pnpm-lock.yaml ./
 # No need to clean cache because production uses standalone build
 RUN pnpm install --frozen-lockfile
 
+ARG BUILD_VERSION
+ENV STUDIO_VERSION="$BUILD_VERSION"
+
```
You want to move this to line 29 so it's available on both dev and production images.

joshenlim replied:
pushed an update :)

joshenlim committed:
Add new env var to turbo.json

joshenlim committed:
Update Dockerfile

sweatybridge committed:
Update publish_image.yml

sweatybridge reviewed (approved):
Docker changes look good!

charislam committed:
fix(docs): typo in authorization header example (#33770)

saltcod committed:
Chore/api keys perms (#33684)

* Dont try and load keys if user doesnt have permission

* Async check perms

mansueli committed:
chore: update stripe webhook example (#33774)

* Chore: Update stripe-webhooks.mdx

* Update index.ts

gregnr committed:
Docs: Automatic embedding updates (#33773)

* docs: add back dimension size to halfvec column

* docs: change headers back to lowercase

charislam committed:
chore: docs linting (#33574)

jordienr committed:
allow nested routes for last visited route in logs (#33775)

allow nested routes

Commit:
chore(docs): update Swift Realtime filter to use new filter type (#33748)

charislam committed:
clean: remove console logs (#33623)

SaxonF committed:
move templates in inline editor (#33759)

saltcod committed:
Chore/column dropdown bug (#33776)

Fix the blasted column dropdown bug

burmecia committed:
docs: add Notion Wasm FDW docs (#33799)

pamelachia committed:
chore: add signin method to tracking (#33757)

* chore: add signin method to tracking

* include unknown when signinmethod unknown

* use useParams

---------

Co-authored-by: Alaister Young <a@alaisteryoung.com>

jordienr committed:
add pgbouncer config and status queries (#33772)

* add queries

* use different keys

* move folder

* rm old files

* Small refactoir

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

encima committed:
remove fly section and add notes to clarify the SB template on Grafana (#33655)

* remove fly section and add notes to clarify the SB template on Grafana

* prettier changes made

kevcodez committed:
chore: hint user on how to reduce disk (#33792)

darora committed:
docs: clarify impact of `--no-restart` flag when changing custom PG config (#33751)

* feat: clarify impact of `--no-restart` flag when changing custom PG config

* docs: fix liniting issues

docs: keep american spelling

---------

Co-authored-by: Tom Ashley <tom.ashley@gmail.com>

alaister committed:
fix: destructive query warning on run button (#33801)

Commit:
fix(docs): replace {} with JSON.stringify({}) in custom send-sms edge function (#33794)

fix(docs): replace {} with JSON.stringify({})

Aryankadyan committed:
docs: enhance GoTrue description in README (#33798)

j4w8n committed:
docs: clarify triggered password recovery events (#33797)

Co-authored-by: Jason Creviston <jason@malynium.com>

Vandivier committed:
docs: specify Sig v4 (#33755)

for an example of the confusion this edit solves, see:
https://github.com/supabase/storage/issues/634#issuecomment-2664456600

tengweiherr committed:
docs(supabase-js): add example to clarify how to fetch data with spaces in table name (#33443)

* docs: add supabase-js example with spaces in table names

* update copy and example

charislam committed:
ci: docs lint prevent prs hanging (#33819)

Conditional and required checks don't mix well on GitHub -- if a required check is skipped it still hangs, preventing PRs from getting merged indefinitely. Instead, we need to to run the action on every PR, but bail early if no docs content changes were made.

Since the logic for when/how this action runs is getting a bit convoluted, I also split up the tests against master (run against all files, no conditional checks) from the tests aginst PRs (run against changed files, with conditional checks)

charislam committed:
feat: llms.txt (#33780)

* feat: llms.txt

* feat: split llms.txt into multiple files

We have too many docs, so the concatenated text file uses an unreasonable amount of tokens. Chunk it up a little so it's more usable.

charislam committed:
fix: docs redirects (#33821)

Redirects weren't working because they were listed in the wrong order: the first listed ones should have higher priority.

fsansalvadore committed:
Add User Impersonation feature (#33820)

add User Impersonation feature

kiwicopple committed:
chore: update cli reference doc (#33827)

kiwicopple committed:
chore: update self-hosted image versions (#33826)

* chore: update image versions for docker/docker-compose.yml

* fix(self-hosted): studio healthcheck

---------

Co-authored-by: Andrew Valleteau <avallete@users.noreply.github.com>

loong committed:
chore: remove event enums (#33730)

* chore: remove event enums

* chore: remove more event enums

* chore: remove unused HomepageHeroRequestDemoClickedEvent

* chore: remove unused HomepageHeroRequestDemoClickedEvent

* chore: rename custom_report events to use underscores

* chore: remove telemetry enum

* chore: remove unused imports

* chore: remove enums from merged events

* chore: remove unused imports

* chore: add typedocs for module and hide exported types that are not events

encima committed:
add inspect report to cli docs (#33815)

MildTomato committed:
Feat/sidebar studio (#33612)

* add sidebar

* sidebar docs added

* add sidebar

* fixed issues

* fix mobile

* move settings icon back down

* remove old stuff

* Delete SideBar.tsx

* use ProfileImage component

* spelling

* Update ThemeSettings.tsx

* Update ThemeSettings.tsx

* Update Sidebar.tsx

* layout udpate

* Update Sidebar.tsx

* fixes

* Minor lints

* Update Sidebar.tsx

* create a alt user dropdown for self hosting and local

* add fragment back in

* Adds Theme Selector for local/self hosting

* Update UserDropdown.tsx

* fix theme selector for local self host

* Update index.ts

* move dropdown and fix

* Update LayoutHeader.tsx

* update const

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

joshenlim committed:
Fix check before constructing new URL in TerminalInstructions (#33836)

saltcod committed:
Edge functions secrets updates (#31329)

* Rename secrets to env vars, add inline form

* Add multiple

* Fix adding multiple

* Change name back

* Sm update

* Add docs for secrets management

* Fix file paths

* Prettier

* Fix image paths

* Images

* clean up

* Fix form submit

* Minor fixes.

* Remove console logs.

---------

Co-authored-by: Ivan Vasilov <vasilov.ivan@gmail.com>

sweatybridge committed:
docs: explain how to order declarative schemas (#33832)

* docs: explain how to order declarative schemas

* Update declarative-database-schemas.mdx

* Update declarative-database-schemas.mdx

* Add prod note

---------

Co-authored-by: Terry Sutton <saltcod@gmail.com>

saltcod committed:
Chore/ef secret reset (#33842)

* Reset secrets form after submission

* Try a different fix.

* Align the delete button.

---------

Co-authored-by: Ivan Vasilov <vasilov.ivan@gmail.com>

fsansalvadore committed:
promote grafana in reports (#33657)

* promote grafana in reports

* add docs link

* lighter svg

* fix telemetry event

* next/image

* unused import

* BASE_PATH

* img

* Apply suggestions from code review to remove STUDIO bit

* add bg to button

* fix telemetry action

---------

Co-authored-by: Jordi Enric <37541088+jordienr@users.noreply.github.com>
Co-authored-by: Pamela Chia <pamelachiamayyee@gmail.com>

saltcod committed:
Chore/efs pasting again (#33847)

fix pasting multiple times

SaxonF committed:
Improve edge functions layout (#33723)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* update edge functions layout

* remove params

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* fix merge errors

* Address comments

* Remove unnecessary prop

* Remove settings/functions and add redirects in next config

* fix empty

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

kiwicopple committed:
blog: pgrouting (#33848)

* adds a new blog post for pgrouting

* fix image

* fix images and titles

* fixing naming

* fixes link

dragarcia committed:
chore: add `AWS_NEW` provider (#33697)

* feat: add AWS_NEW provider

* fix selecting aws region

* chore: add valid instance sizes

---------

Co-authored-by: Alaister Young <a@alaisteryoung.com>

fsansalvadore committed:
docs contributing layout (#33833)

* contributing layout

* contributing layout mobile

* gap

* Update apps/docs/app/contributing/ContributingToC.tsx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* Update apps/docs/app/contributing/ContributingToC.tsx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* add missing menu entry

* remove feedback anchor from toc

* cleanup

* sheet height

* sheet height

* breadcrumb

---------

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

fsansalvadore committed:
Feedback component polish (#33840)

* polish Feedback component

* fix telemetry event

* step to fix animation

* anim orchestration details

* add border

* remove border

* toc gap

* h3

MildTomato committed:
fix sidebar states (#33872)

TheOtherBrian1 committed:
Update connecting-to-postgres.mdx | made example region agnostic (#33853)

saltcod committed:
Chore/move dot (#33882)

Nudge lint dot

SaxonF committed:
Feat/plan upgrade improvements (#33804)

* upgrade dialog changes

* plan upgrade modal

* update dialog improvements

* factor in customer balance

* copy

* add PH a/b test

* fix ts

* smol stuff

* fixes

* group compute fees

* smol edits

* Update SubscriptionPlanUpdateDialog.tsx

* Update SubscriptionPlanUpdateDialog.tsx

* partner billing

* downgrade

* margin

* show pricing info + filter out long tweets

* fix tooltip

---------

Co-authored-by: Kevin Grüneberg <k.grueneberg1994@gmail.com>

saltcod committed:
Chore/theme change (#33879)

Update AIEditor theme to Supabase theme

MildTomato committed:
Fix: sidebar flex shrink issue FE-1435 (#33887)

Update sidebar.tsx

doublethink committed:
Second phase of HSTS header settings (#33857)

apply second phase of hsts header settings

SaxonF committed:
Improve single edge function layout (#33725)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* update edge functions layout

* remove params

* single function layout

* invocation cleanup

* remove vars

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* fix merge errors

* fix merge errors

* rmeove import

* Address comments

* Remove unnecessary prop

* Remove settings/functions and add redirects in next config

* fix empty

* Fix key prop issue and fix selected state in layout for single edge function

* Clean up edge function details

* Use zodResolver

* Refactor file name

* Remove unnecessary fragment

* Small refactors

* Refactor deleting edge function to use ConfirmationModal

* edge details fixes

* Small adjustment to paddings

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

kamilogorek committed:
feat: Show updated_at time for edge secrets (#33716)

SaxonF committed:
Create edge function (#33726)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* update edge functions layout

* remove params

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* create edge functon

* use component

* flag

* fix merge errors

* fix imports

* fix merge issues

* fix import

* fix flag

* remove unused

* move templates

* re-add flag

* Small refactors

* Smol refactors

* Refactor

* add rename and delete via context

* create function fixes

* moved function name

* Smol update

* Small fix for selected state in edge functions side menu when on /new

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

saltcod committed:
Chore/update lints (#33749)

Update lints

SaxonF committed:
Edge functions empty state (#33802)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* update edge functions layout

* remove params

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* create edge functon

* use component

* flag

* fix merge errors

* fix imports

* fix merge issues

* fix import

* fix flag

* remove unused

* simple empty

* empty state improvements

* move templates

* re-add flag

* use templates

* flag

* copy functions

* feature flag it

* Fix issues and clean up

* Smol fix

* Add comment

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

fsansalvadore committed:
pgrouting graph thumb (#33894)

ivasilov committed:
fix: Remove the keybind for opening/closing the sidebar (#33896)

Remove the keybind for opening/closing the sidebar.

charislam committed:
docs: update troubleshooting for invalid claim (#33884)

invalid cliam: missing sub seems to (also?) be returned as a 403, not a 401. Discovered this when I was testing something else, so updating the troubleshooting guide for it.

charislam committed:
chore(docs): add spelling exception (#33818)

Presigned/presigning should be allowed

ssiyad committed:
doc: postgres/row-level-security: wrong link to auth (#33900)

charislam committed:
chore: add preferred usage item for storage docs (#33597)

chore(docs lint): add preferred usage for storage connections

charislam committed:
fix: troubleshooting sync (#33908)

Relative links don't make sense in GitHub discussions so they need to be converted to absolute links for syncing.

laurenceisla committed:
docs: add more filtering examples to the API docs (#33899)

- Add logical operators "not" and "or"
- All Javascript examples are now present for Bash

Co-authored-by: Terry Sutton <saltcod@gmail.com>

pamelachia committed:
chore: clarify bursting and recommend compute (#33823)

joshenlim committed:
Support for Dedicated Pooler in Connection Pooling (#33817)

* Init

* Initial set up for hooking up supavisor and pgbouncer

* Hook up pgbouncer status check after swapping pooler type

* Add check for nano compute for switching to pg bouncer

* Add check for ipv4 addon

* Remove expect error tag

* Add badge to select options for pooler types

* Remove statement mode

* Resolve undefined problem with react hook form

* Fix

* Update UI texts from PgBouncer to Dedicated Pooler

* Feex

* FEEX

* Fix

* Small update to UI

* Smol update

kevcodez committed:
chore: increase stale time for project addons / org subscription (#33911)

Occasionally results in a 429 in API - there are plans to fix this on the API level, but it's not a quick change, so hoping to reduce the likeliness of 429 with a higher stale time. Subscriptions and project addons rarely change and if they are changed, it's invalidated on the client anyway.

joshenlim committed:
Use TimestampInfo for scheduled backups (#33892)

* Use TimestampInfo for scheduled backups

* Fix

* Update TimestampInfo

* Fix

* Fix

SaxonF committed:
Test edge functions (#33728)

* page components

* page component changes

* settings but broken saving

* rvert

* use sheet for provider

* styling

* remove things

* Some refactoring and fixing, added JSDocs to layouts

* Smol refactor

* Fix

* Update JSDocs

* updated scaffolding

* update edge functions layout

* remove params

* single function layout

* invocation cleanup

* remove vars

* Clean up

* Spelling

* Clean up FormFieldWrappers

* One last clean up

* test edge function

* sheet flag

* remove prop

* fix merge errors

* fix merge errors

* update sheet

* rmeove import

* fix ts

* Some clean ups

* Fix

* Make test response area resizeable

* Final clean up

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

joshenlim committed:
Support for Dedicated Pooler in Connection Pooling Part 2 (#33829)

* Init

* Initial set up for hooking up supavisor and pgbouncer

* Hook up pgbouncer status check after swapping pooler type

* Add check for nano compute for switching to pg bouncer

* Add check for ipv4 addon

* Remove expect error tag

* Update copy in IPv4SidePanel

* Add badge to select options for pooler types

* Hook up pgbouncer config for connect UI

* Refactor pooling-configuration react queries to supavisor-configuration

* Update Ipv4 compatability UI indicators in Connect UI when on pgbouncer

* Remove statement mode

* Resolve undefined problem with react hook form

* Fix

* Update UI texts from PgBouncer to Dedicated Pooler

* Feature flag changes

* Add pooler settings link in Connect UI

* Smol update

* Update session pooler description for pgbouncer

MildTomato committed:
fix: handle position (#33913)

Update ProjectLayout.tsx

joshenlim committed:
Fix connection pooling feature flag (#33914)

* Fix connection pooling feature flag

* Fix type issues with lint-query after updating API typs

* Apply same fix for connection_string to ConnectTabContent

* Update useMemo dependency arrays in ConnectionPooling

* Remove allowPgBouncerSelection enabled check on react queries - feels unnecessary and adds some complexity to the loading state of the forms

* Remove conosle log

ekhar committed:
Fix Wasm Edge Function Docs (#33910)

* fixed docs bug: when importing wasm, you need to include the js and use this. Otherwise, there will be wasm instantiate errors

* Update wasm.mdx

Got rid of filename on top of markdown

* Update index.ts

got rid of filename comment at the top

pidanou committed:
docs: edit self-host guide (#32695)

- Added information to make the studio work outside of localhost. This information is only given inside .env.example

joshenlim committed:
Bump configcat refresh rate (#33954)

alaister committed:
[FE-1426] feat(studio): clerk fdw (#33803)

* feat(studio): clerk fdw

* prettier

* docs: add Clerk FDW docs

* Smol update

---------

Co-authored-by: Bo Lu <lv.patrick@gmail.com>
Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

SaxonF committed:
Link to auth provider (#33951)

* link to provider

* Simplify implementation

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

ivasilov committed:
chore: Bump vulnerable deps (#33915)

* Consolidate different versions of postcss. Remove @mertasan/tailwindcss-variables from ui package.

* Bump vitest in pg-meta to remove a vulnerable version of vite.

* Bump dompurify.

* Bump all octokit deps to solve 2 vulns.

* Fix warnings about destructuring json imports.

* Remove a console.log in docs.

* Fix the changelog page to use an ESM import.

* Stabilize the generate-sitemap script output.

* Try another fix for the changelog.

encima committed:
Update metrics documentation (#33893)

* add genera deployment instructions for both grafana and prometheus

* add supabase-grafana to spellchecks

joshenlim committed:
Add support for configuring realtime settings (#33875)

* Add support for configuring realtime settings

* Refactor

* Fix TS

* Fix TS

* Smol update

* Hide new UI for local selfhosted

* Add feature flag

* Fix reset logic

* Fix

* minor refactor

* Fix package.json

* Simplify

joshenlim committed:
Fix package-lock (#33968)

* Fix package-lock

* Set the dompurify to 2.5.8.

---------

Co-authored-by: Ivan Vasilov <vasilov.ivan@gmail.com>

avallete committed:
chore(docs): setup docs for overrideTypes (#33705)

* chore(docs): setup docs for overrideTypes

* fix: add overrideTypes to docs section

* chore: regen specs

* chore: update snapshot

* chore: regenerate all specs

* revert realtime spec changes

---------

Co-authored-by: Charis Lam <26616127+charislam@users.noreply.github.com>

clarencepenz committed:
chore: updated Developers.md (#33933)

* chore: change Git url from http --> https

* chore: change nodejs url from http --> https

vuksanv committed:
Laravel documentation addition to explain schema needs to be created (#33973)

* Add a note that schema needs to exist on Supabase before running
migrate. Otherwise one will receivie a following error

$ php artisan migrate

   INFO  Preparing database.

  Creating migration table ........................................................................................................... 599.73ms FAIL

   Illuminate\Database\QueryException

  SQLSTATE[3F000]: Invalid schema name: 7 ERROR:  no schema has been selected to create in at character 14 (Connection: pgsql, SQL: create table "migrations" ("id" serial not null
primary key, "migration" varchar(255) not null, "batch" integer not null))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:823
    819▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    820▕                 );
    821▕             }
    822▕
  ➜ 823▕             throw new QueryException(
    824▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    825▕             );
    826▕         }
    827▕     }

      +41 vendor frames

  42  artisan:16
      Illuminate\Foundation\Application::handleCommand()

* Apply suggestions from code review

---------

Co-authored-by: Vladimir Vuksan <vuksan-github@vuksan.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

charislam committed:
feat: support typedoc 0.27 (#33905)

Type docs pulled from JS client libraries were expected to be in version 0.22, which unfortunately is a bit old and incompatible with TypeScript 5. Therefore any JS lib that upgrades their toolchian to TS 5 will inadvertently break their type specs in docs.

This PR updates docs to also support 0.27 (the latest stable version, compatible with TS 5), in addition to the old version. Parallel support is required since many libs are still on the old version.

The main changes are:
- The comment shape has changed in typedoc.
- kindString has been deprecated in favor the kind numbers.
- indexSignatures are now plural (an array).

dshukertjr committed:
docs: Update MCP docs for connecting to Cursor (#33952)

* docs: Update MCP docs for connecting to Cursor

* minor fix

SaxonF committed:
Update authentication advanced page (#33955)

* init PrePostTab component

* init

* support auto fixing rounded of inputs/select/textarea

* remove form item layout prepost

* Update FormLayout.tsx

* Update FormLayout.tsx

* Update FormLayout.tsx

* form layout and advanced

* Update advanced.tsx

* protection and hooks

* fix

* parse

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>

SaxonF committed:
Updating authentication email pages (#33958)

* init PrePostTab component

* init

* support auto fixing rounded of inputs/select/textarea

* remove form item layout prepost

* Update FormLayout.tsx

* Update FormLayout.tsx

* Update FormLayout.tsx

* form layout and advanced

* Update advanced.tsx

* protection and hooks

* auth emails

* preview styles

* fix

* parse

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>

joshenlim committed:
Hide pool mode field in connection pooling (#33962)

* Hide pool mode field in connection pooling

* Add comment

sayil committed:
Updated link to management API documentation (#33982)

* Updated link to management API documentation

* add permanent redirect

---------

Co-authored-by: Greg Papas <gregpapas@Gregs-MacBook-Pro.local>
Co-authored-by: Wen Bo Xie <wenbo.xie3@gmail.com>

kevcodez committed:
docs: example API URL should point to localhost (#33986)

Staging API is not accessible from localhost

fsansalvadore committed:
fix link (#33967)

* fix link

* pnpm i

avallete committed:
chore(pg-meta): migrate table-privileges tests (#33506)

* chore(pg-meta): port functions tests

* chore: add new test for CURRENT arg creation

* chore: revert query style change

* chore: restore PGFunction type inference

* wip: remove type cast int8 on arg_types

* Revert "wip: remove type cast int8 on arg_types"

This reverts commit 6878a93bfbc8c4a87f5dd36d5a1ed13381760d85.

* wip: make tests work with both clients

* chore: use pg and type parser

* chore: use only pg

* chore: migrate table-privileges tests

staaldraad committed:
feat: SOC 2 docs page (#33965)

* feat: add SOC 2 page to docs

* feat: more linking

* fix: make linters happy

* fix: prettier

* chore: make security page available

* typo: sp

* publish security

* chore: publish new security section

* change mobile menu to show Security on Security section

---------

Co-authored-by: Charis Lam <26616127+charislam@users.noreply.github.com>

avallete committed:
chore(pg-meta): migrate column-privileges (#33643)

chore(pg-meta): migrate column-privilges

avallete committed:
chore(pg-meta): migrate indexes (#33645)

* chore(pg-meta): migrate indexes

* chore(pg-meta): fix vector to array conversion

* chore: use common naming

Hallidayo committed:
www: storage example code change (#33054)

storage example code change

avallete committed:
chore(pg-meta): migrate version (#33648)

avallete committed:
chore(pg-meta): migrate types sql (#33649)

avallete committed:
chore(pg-meta): migrate triggers (#33675)

avallete committed:
chore(pg-meta): migrate policies (#33680)

fsansalvadore committed:
customer story asap work (#33873)

* asap.work customer story

* add author

fsansalvadore committed:
meshy customer story (#33876)

Co-authored-by: Prashant Sridharan <914007+CoolAssPuppy@users.noreply.github.com>

alaister committed:
fix(studio): whitelist duplicate name error on project creation (#33983)

fsansalvadore committed:
Resend customer story (#33746)

* resend customer story

* founded location

---------

Co-authored-by: Prashant Sridharan <914007+CoolAssPuppy@users.noreply.github.com>

EstebanDalelR committed:
Fix broken line (#34002)

Line was split in two unnecessarily.

doublethink committed:
Added hsts preload directive to vercel config (#33980)

SaxonF committed:
Updating authentication mfa and url pages (#33956)

* init PrePostTab component

* init

* support auto fixing rounded of inputs/select/textarea

* remove form item layout prepost

* Update FormLayout.tsx

* Update FormLayout.tsx

* Update FormLayout.tsx

* form layout and advanced

* Update advanced.tsx

* protection and hooks

* mfa and url

* remove onchange

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>

SaxonF committed:
Updating authentication rate limits and session pages (#33957)

* init PrePostTab component

* init

* support auto fixing rounded of inputs/select/textarea

* remove form item layout prepost

* Update FormLayout.tsx

* Update FormLayout.tsx

* Update FormLayout.tsx

* form layout and advanced

* Update advanced.tsx

* protection and hooks

* rate limits and sessions

* remove default value

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>

staaldraad committed:
feat: add links to product security (#34000)

* feat: add links to product security

* chore: add link for product-security

* fix: spelling

* fix: spelling

* Update apps/docs/content/guides/security/product-security.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

---------

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

staaldraad committed:
chore: hipaa responsibilities doc (#33996)

* feat: add HIPAA responsibility docs

* chore: additional config guidance

* chore: mention geo specific status of HIPAA

* fix: spelling

* chore: faq and soc 2 comparisons

* chore: better linking to shared responsibility

* fix: spelling

* Update apps/docs/content/guides/security/hipaa-compliance.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* Update apps/docs/content/guides/security/hipaa-compliance.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* Update apps/docs/content/guides/security/hipaa-compliance.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

---------

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

alaister committed:
fix: bytea null values (#34014)

* fix: bytea null values

* fix null values in foreign row selector

* sort imports

joshenlim committed:
Update UI of the version popover

joshenlim committed:
fix

joshenlim committed:
Merge master resolve conflict
